src/main/java/bio/terra/service/dataproject/DataLocationService.java

```diff
@@ -44,17 +47,30 @@
     private final DataLocationSelector dataLocationSelector;
     private final GoogleResourceService resourceService;
     private final ProfileService profileService;
+    private final SamConfiguration samConfiguration;
 
     @Autowired
     public DataLocationService(
             DataProjectDao dataProjectDao,
             DataLocationSelector dataLocationSelector,
             GoogleResourceService resourceService,
-            ProfileService profileService) {
+            ProfileService profileService,
+            SamConfiguration samConfiguration) {
         this.dataProjectDao = dataProjectDao;
         this.dataLocationSelector = dataLocationSelector;
         this.resourceService = resourceService;
         this.profileService = profileService;
+        this.samConfiguration = samConfiguration;
+    }
+
+    public Map<String, List<String>> getStewardPolicy() {
+        //hard code user string for now
+        String role = "roles/bigquery.jobUser";
```

Comment:
Can you move this to a constant up top like we did with `DATA_PROJECT_SERVICE_IDS`? Maybe `BQ_JOB_USER_ROLE`?